.github/workflows/dashboard.yml

```diff
@@ -0,0 +1,71 @@
+name: Token Dashboard
+
+on:
+  pull_request:
+    branches: [ master ]
+    paths:
+    - solidity/dashboard/**
+
+# For pull_request_target:
+#  - Checkout with repository set to PR repo, ref same (see
+#    https://github.com/actions/checkout#usage).
+#  - Trigger from issue comment?
+
+jobs:
+  build-and-deploy:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [12.11.x]
+
+    if: startsWith(github.head_ref, 'releases/mainnet/token-dashboard/')
+    defaults:
+      run:
+        working-directory: solidity/dashboard
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v2
+      with:
+        node-version: ${{ matrix.node-version }}
+    - run: npm ci
+    #- run: npm run lint
+    #- if: github.event_name == 'push' TODO uncomment when mainnet builds happen this way
+    #  run: npm run build --if-present
+    - if: github.event_name == 'pull_request'
+      run: npm run build
```

Comment:
Is this specific to `mainnet` releases that we build it with `npm` and not use docker as in the current Circle CI config for keep-dev and keep-test?

Reply:
Well, docker uses npm internally, but it's actually specific to the preview build right now. I want to slowly pull docker + kube out of the deployment process for anything that's just serving static assets---they're just needless complexity.